File: FoodToOrderFrontend/src/app/models/cartdish.ts
import { Carts } from "./cart"
import { Dishes } from "./dishes"

export class CartDish{
    c_id: string
    d_id: string
    dish?:Dishes = new Dishes('','',0,'','',true)
    quantity: number
    //cart:Carts = new Carts('',0,'',[],[],[])
    
    
    constructor(cid:string,did:string,qty:number){
        this.c_id=cid
        this.d_id=did
        this.quantity =qty
    }
}

File: FoodToOrderBackend/FoodToOrderApi/FoodToOrderDB/CartDish.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FoodToOrderDB
{
    public class CartDish
    {
        public int? c_id {  get; set; }
        public Cart? Cart { get; set; }
        public int? d_id { get; set; }
        public Dish? dish { get; set; }
        public int quantity { get; set; }

    }
}
